### DISCUSSION TITLE
Advice needed - co-existence of the 1.5x and 2.0

### DISCUSSION THREAD

**Discussion by prandeamus** [SpenceKonde/ATTinyCore#744] [Development Plans]
Sorry if this is "obvious" to Arduino and ATTiny long-term users. If I want to have the current "production" version 1.5x and the in-development 2.x branch installed at the same time, what's the best (or least bad) way to do it. I'm developing a simple 85/88 project at the moment, and I'd like to use 2.x to try it out, but if something goes wrong I might want to check back to to 1.5x to see if it's my fault or an unintended error in 2.x or whatever.

You can assume I know how to use git and check out the right branch and files, but when it comes to telling Arduino to use one version or the other, what's a fairly straightforward to have them on the local file system and switch between them? 

Is there a versioned URL  for 2.x, i.e. something like http://drazzy.com/package_drazzy.com_index.json that would download v2 automatically in parallel?

Sorry if this is documented somewhere and I missed it.

**Comment by prandeamus** [comment]
I'm also using Arduino IDE 2.0.3 if that helps, but I also have the older IDE version installed. I'm happy with either one for this project.

**Comment by SpenceKonde** [comment]
We have had problems reported on 2.0.3 that are due to bugs in the new IDE version, I can't recommend that version for any purposes. 

I would use the .zip package to create a "portable" Arduino 1.8.13 installation (you create a folder named portable inside the arduino folder before the first time you start it). Install ATTinyCore 1.5.2 via board manager there. 

Create a nonportable installation (with no version of ATTinyCore installed via board manager), then create a "hardware" folder inside your sketchbook folder (C:/users/yourname/documents/Arduino). Using git or the github desktop tool to clone this repo to the hardware folder.
